Public/css/app.less

```diff
@@ -315,3 +315,13 @@ footer {
 #input-gitignore::-ms-expand {
     display: none;
 }
+
```

Comment:
Also, why do we have two similar CSS files? app.css and app.less?

Reply:
This does not strictly relate to the PR though

Reply:
There was no docs for this but I think LESS was used as a preprocessor, so I added docs for this. Thanks for noticing!